packages/2d/src/curves/SerializedKnot.ts

```diff
@@ -0,0 +1,10 @@
+import {Vector2} from '@motion-canvas/core/lib/types';
+
+export type KnotAutoHandles = {start: number; end: number};
+
+export interface SerializedKnot {
```

Comment:
I don't think `SerializedKnot` is a good name since this representation contains instances of other classes.
I'd go for `ParsedKnot` or `KnotInfo`

Reply:
Agreed. I renamed it to `KnotInfo`